packages/shorebird_cli/lib/src/commands/promote_patch_command.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:collection/collection.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/code_push_client_wrapper.dart';
+import 'package:shorebird_cli/src/config/config.dart';
+import 'package:shorebird_cli/src/deployment_track.dart';
+import 'package:shorebird_cli/src/extensions/arg_results.dart';
+import 'package:shorebird_cli/src/logger.dart';
+import 'package:shorebird_cli/src/shorebird_command.dart';
+import 'package:shorebird_cli/src/shorebird_env.dart';
+
+/// {@template promote_patch_command}
+/// Promotes a patch to the production channel.
+/// {@endtemplate}
+class PromotePatchCommand extends ShorebirdCommand {
+  /// {@macro promote_patch_command}
+  PromotePatchCommand() {
+    argParser
+      ..addOption(
+        'flavor',
+        help: 'The product flavor to use when building the app.',
+      )
+      ..addOption(
+        'release-version',
+        help: 'The release being patched',
+        mandatory: true,
```

Comment:
not required at the moment but would be nice to make these optional and prompt if they aren't specified just to be consistent with the other commands.